packages/eio-ssl/eio-ssl.0.2.0/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+synopsis: "OpenSSL binding to EIO"
+maintainer: ["Antonio Nuno Monteiro <anmonteiro@gmail.com>"]
+authors: ["Antonio Nuno Monteiro <anmonteiro@gmail.com>"]
+license: "LicenseRef-LGPL-WITH-OpenSSL-linking-exception"
+homepage: "https://github.com/anmonteiro/eio-ssl"
+bug-reports: "https://github.com/anmonteiro/eio-ssl/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "ocaml" {>= "5.0"}
+  "ssl" {>= "0.5.13"}
+  "eio_main" {>= "0.7"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/melange-re/melange-compiler-libs.git"
```

Comment:
do you mean the following?
```suggestion
dev-repo: "git+https://github.com/anmonteiro/eio-ssl.git"
```
(it seems that the upstream might need to be updated as well
https://github.com/anmonteiro/eio-ssl/blob/5927222848f981235b658e026183a5b606a379f8/dune-project#L14-L15 )

Reply:
Clearly copied from somewhere else. Nice catch.